doc/source/analytics/log_level.md

```diff
@@ -1,21 +1,33 @@
-# Seldon Core Analytics
+# Logging and log level
 
 Seldon core metrics containers are able to provide different log levels. 
+By default the containers come out of the box with `WARNING` as default log
```

Comment:
As we are mixing different languages (Python and GO) it may be a nice idea to add a simple table that compares different log levels?

Reply:
Also if k8s log levels are integer-denoted it would be nice to make it clear to which e.g. Python log level it corresponds.

Reply:
The problem is that K8s only have level `0` and `1`. You can technically use any level (the higher the level the more verbose it is), but they recommend sticking to those. Thus there is no 1:1 mapping to the Python and Java levels.

But I agree, maybe we could add a table with the levels of each one.

Reply:
I think main benefit from having a table would is to make clear to the reader that from four levels in Python:
`DEBUG`, `INFO`, `WARNING` or `ERROR`
we assign Go levels as
- Warning or Error -> 0
- Info or Debug -> 1

Reply:
I've separated `SELDON_LOG_LEVEL` and `SELDON_DEBUG`, so now that it's more clear I think we probably don't need a table anymore. Mainly because the interface between Python and Go is now the same (i.e. `SELDON_LOG_LEVEL` will just change the logging level).